apollo-runtime/build.gradle.kts

```diff
@@ -1,13 +1,14 @@
 plugins {
   `java-library`
+  kotlin("jvm")
 }
 
 dependencies {
-  add("api", project(":apollo-api")) // apollo-espresso-support uses some apollo-api internals
-  add("api", project(":apollo-http-cache-api"))
-  add("api", groovy.util.Eval.x(project, "x.dep.okHttp.okHttp"))
-  add("compileOnly", groovy.util.Eval.x(project, "x.dep.jetbrainsAnnotations"))
-  add("implementation", groovy.util.Eval.x(project, "x.dep.cache"))
+  api(project(":apollo-api"))
+  api(project(":apollo-http-cache-api"))
+  api(groovy.util.Eval.x(project, "x.dep.kotlin.stdLib"))
+  api(groovy.util.Eval.x(project, "x.dep.okHttp.okHttp"))
+  implementation(groovy.util.Eval.x(project, "x.dep.cache"))
```

Comment:
So we don't need the `jetbrainsAnnotations` at compileTime anymore ?

Reply:
Once we add Kotlin we don't need them anymore. They're actually transitively added already.

Reply:
Could we remove the `annotations` part in the README.md ?https://github.com/apollographql/apollo-android/blame/master/README.md#L53

This has always confused me.